### PR TITLE
[PHP] Update chubbyphp to 3.0

### PR DESCRIPTION
**Pull request by dominikzogg**
----
Closes #3036

### CODE REVIEW

File: php/chubbyphp-roadrunner/composer.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "require": {
-    "chubbyphp/chubbyphp-framework": "~2.8.0",
+    "chubbyphp/chubbyphp-framework": "~3.0.0",
+    "chubbyphp/chubbyphp-framework-router-fastroute": "~1.0",
```
Comment (waghanza) [inline php/chubbyphp-roadrunner/composer.json:4]: do we need to specify version ?

File: php/chubbyphp/composer.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "require": {
-    "chubbyphp/chubbyphp-framework": "~2.8.0",
+    "chubbyphp/chubbyphp-framework": "~3.0.0",
+    "chubbyphp/chubbyphp-framework-router-fastroute": "~1.0",
```
Comment (waghanza) [inline php/chubbyphp/composer.json:4]: do we need to specify version ?

File: php/chubbyphp-roadrunner/composer.json (hunk 1 above)
Comment (dominikzogg) [reply php/chubbyphp-roadrunner/composer.json]: no and fastroute is now also not required as an explicit dependency anymore

File: php/chubbyphp/composer.json (hunk 2 above)
Comment (dominikzogg) [reply php/chubbyphp/composer.json]: no and fastroute is now also not required as an explicit dependency anymore

**Comment by dominikzogg** [comment]
should be ready to merge, if its also ok for you

File: php/chubbyphp-roadrunner/composer.json (hunk 1 above)
Comment (waghanza) [reply php/chubbyphp-roadrunner/composer.json]: could we then use an `*` to force latest version ?

File: php/chubbyphp/composer.json (hunk 2 above)
Comment (waghanza) [reply php/chubbyphp/composer.json]: could we then use an `*` to force latest version ?

Comment (dominikzogg) [reply php/chubbyphp/composer.json]: already did that: https://github.com/the-benchmarker/web-frameworks/pull/3037/commits/6dd75eb1100e20bf17610dc1cbcfe5be3f221b17

File: php/chubbyphp-roadrunner/composer.json (hunk 1 above)
Comment (dominikzogg) [reply php/chubbyphp-roadrunner/composer.json]: already did that: https://github.com/the-benchmarker/web-frameworks/pull/3037/commits/6dd75eb1100e20bf17610dc1cbcfe5be3f221b17

Comment (waghanza) [reply php/chubbyphp-roadrunner/composer.json]: Ok, UI tricks me 😚

File: php/chubbyphp/composer.json (hunk 2 above)
Comment (waghanza) [reply php/chubbyphp/composer.json]: Ok, UI tricks me 😚

**Comment by dominikzogg** [comment]
@waghanza who is in charge to merge it?

**Comment by waghanza** [comment]
me @dominikzogg ;-) will do after `CI` passes :stuck_out_tongue_closed_eyes: 

and we will have fresh results tomorrow :tada:

**Comment by dominikzogg** [comment]
@waghanza thx

**Pull request merged**